db/schema.rb

```diff
@@ -38,20 +38,20 @@
     t.index ['post'], name: 'index_likes_on_post'
     t.index ['post_id'], name: 'index_likes_on_post_id'
     t.index ['user'], name: 'index_likes_on_user'
-    t.index ['author_id'], name: 'index_likes_on_author_id'
+    t.index ['author_id'], name: 'index_likes_on_user_id'
   end
 
   create_table 'posts', force: :cascade do |t|
     t.datetime 'created_at', null: false
     t.datetime 'updated_at', null: false
     t.string 'title'
     t.string 'text'
-    t.integer 'commentsCounter'
-    t.integer 'likesCounter'
+    t.integer 'comments_counter'
+    t.integer 'likes_counter'
     t.bigint 'author_id', null: false
     t.string 'user'
```

Comment:
- [ ] @yosefHesham as requested by the previous reviewer is difficult to run your app locally because of these errors. If you check the Entity Relationship Diagram that was given to us you'll discover that there's no column in the user table called `user`. I'll kindly advise you to restructure your db and models.
- [ ] Also check the likes and posts table, the same error. Your comments table contain unknown columns that are not present in this [Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md)